### DISCUSSION TITLE
Feature Request: Absolute Accuracy for osu!mania

### DISCUSSION THREAD

**Discussion by PythonTryHard** [ppy/osu#12989] [Ideas]
I apologize if this is inappropriate. If so, I'd gladly take criticism.

After some practicing on 7K mania, an idea struck me about accuracy. Since mania differs from the remaining three modes for being easier on players in terms of accuracy ([300\*, 300, 200, 100, 50] vs [300, 100, 50]), and 1000000 Perfect Clear can only be achieved through clearing the map with nothing but 300\*, why not implement a new accuracy meter, which I came up at the time with the name "Absolute Accuracy"? Similar to Cytus, where a 1000000 (MM) meaning can still have less than 100% TP since in Cytus there are also two types of Perfect: Black Perfect and Rainbow Perfect (if I recall correctly on the name) respectively corresponding to osu!mania 300 and 300\*.

Thank you for spending your valuable reading through a rank #37k mania player's erratic idea, and I apologize for any mistake, whether technical, grammatical or typos I made if any.

**Comment by Rekkonnect** [comment]
The scoring algorithm is currently implemented so that 1,000,000 is only achievable if the player only gets 300g in the entire map. As planned, a new scoring algorithm will be implemented for the purposes of matching and balancing pp.

Since a 100% accuracy (SS) play can still have normal 300s, instead of only 300gs, obviously the most accurate plays would have to be more worthy and valuable, thus having a greater score. Therefore I personally think it would be best to have such scoring differences between 100% accuracy plays.
